tests/test_generator.py

```diff
@@ -11,16 +11,16 @@ def test_generator() -> None:
     # data_type = data_file.stem.split("_")[0]
     # assert data_type == "drone"
     # location_file = Path("tests/test_data/drone_test_files.csv")
-    # # Create the STAC catalog.
-    # os.environ["STAC_API_URL"] = "placeholder"
+    # # Create the Stac catalog.
+    # os.environ["Stac_API_URL"] = "placeholder"
```

Comment:
Lets not allow the linter or whatever changed this to decapitalize then env variable names. The comments should not have changed either but its less important.

Reply:
A lot of the refactoring was done cause I wanted to check if the CI works well. I will reconfigure the linter to not raise an error for this.